pombola/feedback/views.py

```diff
@@ -14,19 +16,24 @@ def add(request):
 
     submit_was_success = False
     return_to_url      = None
+    submit_error_message  = None
 
     # If it is a post request try to create the feedback
     if request.method == 'POST':
         form = FeedbackForm( request.POST )
+        recaptcha_response = request.POST.get("g-recaptcha-response", None)
+
         if form.is_valid():
             feedback = Feedback()
             feedback.url      = form.cleaned_data['url']
             feedback.email    = form.cleaned_data['email']
             feedback.comment  = form.cleaned_data['comment']
 
-            # if there is any content in the honeypot field then label this comment as spammy
-            if form.cleaned_data['website']:
-                feedback.status = 'spammy'
+            submit_was_success = True
+
+            if not recaptcha_client.verify(recaptcha_response):
```

Comment:
We should check for `settings.GOOGLE_RECAPTCHA_SECRET_KEY` before calling `recaptcha_client.verify` so that the submit will be a success if  `settings.GOOGLE_RECAPTCHA_SECRET_KEY` is not present.